### DISCUSSION TITLE
HomeAssistant (Hassio / HA) Add-ons

### DISCUSSION THREAD

**Discussion by arieltools** [tagyoureit/nodejs-poolController#667] [General]
https://github.com/arieltools/njs-pc-ha-addons

One-click install add-ons that lets you add these two projects directly to your HomeAssistant box.

A good counterpart to related project - https://github.com/tagyoureit/nodejs-poolController/discussions/559.

![image](https://user-images.githubusercontent.com/18470870/209573904-bdc3c8eb-423c-4d8d-92df-40e1ca81e24e.png)

![image](https://user-images.githubusercontent.com/18470870/209573928-4b1392c9-8884-4fd2-9334-a2f4c3c03a49.png)

**Comment by rstrouse** [comment]
Looking forward to giving them a shot.
